### DISCUSSION TITLE
Blur Effect works on Mac but doesnt in windows/android

### DISCUSSION THREAD

**Discussion by NykuluzM** [enisn/UraniumUI#650] [Q&A]
TLDR: Im still a beginner so my code is still messy 
code:  <ScrollView>
        
        <toolkit:UniformItemsLayout MaxColumns="2" HorizontalOptions="Center" >
            <Border HeightRequest="120" WidthRequest="120" Margin="15"
                        Stroke="{StaticResource grad}" StrokeThickness="2">
                        <Border.StrokeShape>
                            <RoundRectangle CornerRadius="40,40,40,40" />
                        </Border.StrokeShape>
        <Grid>
        <Image x:Name="ProfileImageBlurred" IsVisible="False" Aspect="AspectFill">
            <Image.Effects>
                <uranium:BlurEffect Mode="Dark" AccentOpacity="0.8" AccentColor="Purple" />
            </Image.Effects>
        </Image>    
        <Image x:Name="ProfileImage" Aspect="AspectFill" IsVisible="True">
            
        </Image>
        <ImageButton x:Name="editProfile"  IsVisible="False" Clicked="SelectImageButton_Clicked" WidthRequest="40"
              Source="profileedit.png" Background="Transparent"  VerticalOptions="Center" HorizontalOptions="Center" Margin="0,0,0,0">
                
        </ImageButton>
       
        </Grid> 
            </Border>
            <VerticalStackLayout WidthRequest="240"  HeightRequest="51" Padding="40,0,0,0"> 
                <Label Text="carlslasieer1@gmail.com"  FontSize="12" MaxLines="1" LineBreakMode="NoWrap"  TextColor="White"/>
                <Label Text="dnXBDX3Nx1diekwU53n9L8aOaTz1" FontSize="12" MaxLines="1" LineBreakMode="NoWrap"  TextColor="White"/>
            </VerticalStackLayout>
        <Button Text="Unblur Image" IsVisible="True" Clicked="UnBlurImageButton_Clicked" WidthRequest="120"
                VerticalOptions="End" HorizontalOptions="Center" Margin="0,0,0,0" />
        <Button Text="Blur Image" IsVisible="True" Clicked="BlurImageButton_Clicked" WidthRequest="120"
                VerticalOptions="End" HorizontalOptions="Center" Margin="0,0,0,0" />
        </toolkit:UniformItemsLayout>
   
        
    </ScrollView>

**Comment by NykuluzM** [comment]
private void BlurImageButton_Clicked(object sender, EventArgs e)
{
    ProfileImageBlurred.IsVisible = true;
    ProfileImage.IsVisible = false;
    editProfile.IsVisible = true;
}
private void UnBlurImageButton_Clicked(object sender, EventArgs e)
{
    ProfileImageBlurred.IsVisible = false;
    ProfileImage.IsVisible = true;
    editProfile.IsVisible = false;  
}
